src/main/java/com/booster/cliclient/CreateVocabularyEntryInput.java

```diff
@@ -0,0 +1,12 @@
+package com.booster.cliclient;
+
+import lombok.Builder;
+import lombok.Data;
+
+// todo: consider creating a separate 'api' project that contains all API contracts
+@Data
+@Builder
```

Comment:
Let`s incline to the style of using only necessary annotations and @Accessor(chain=true) instead of @Builder.

Reply:
Fixed